docs/DeveloperGuide.md

```diff
@@ -1232,53 +1232,33 @@ Given below are the planned enhancements. The _current behaviour_ specifies how
-
-* More user-friendly info message for `edit` command.
-  * Current behaviour: Upon a successful `edit` command, the info message is rather verbose. For example, with command `edit 1 /phone 12345 /index 2`, (depending on your contact list) the info message might be: `Edited Person: Nguyen; Phones: [12345678, 12345]; Emails: []; Links: []; Courses: []; Specialisations []; Tags: `.
-  * Enhanced behaviour: The info message can be shortened to show the information being changed. For example, with command `edit 1 /phone 12345 /index 2`, the info message can be: `Changed phone at index 2 of the phone list of the contact at index 1 (name: Nguyen) from "87654321" to "12345".`
-
-* More user-friendly info message for `delete` command.
-  * Current behaviour: Upon a successful `delete` command that delete a field from a contact, the info message is rather verbose. For example, with command `delete 1 /phone /index 2`, (depending on your contact list), the info message might be: `Deleted some information of person: Alex Yeoh; Phones: [12345, 1234567, +65 123]; Emails: [alexyeoh@example.com]; Links: [github.com]; Graduation: AY2016/2017 Semester 1; Courses: [Information Systems]; Specialisations []; Tags: [friends]; Priority: Low`.
-  * Enhanced behaviour: The info message can be shortened to show only information being deleted. For example, with command `delete 1 /phone /index 2`, the info message can be: `Deleted phone "12345678" at index 2 of the phone list of the contact at index 1 (name: Alex Yeoh).`
+* More user-friendly info message.
+  * Current behaviour: Upon a successful command dealing with one contact, the info message shows all the information of the contact, which is rather verbose. For example, with command `create /name Nguyen /phone 12345678 /phone 87654321`, the info message is: `Noted, created new contact: Nguyen; Phones: [12345678, 87654321]; Emails: []; Links: []; Courses: []; Specialisations: []; Tags: `.
+  * Enhanced behaviour: The info message can be shortened to show only information being add/edited/deleted. For example, with command `create /name Nguyen /phone 12345678 /phone`, the info message can be: `Noted, created new contact: Nguyen; Phones: [12345678, 87654321].`
 
 ### Model
 
-* More relaxed person uniqueness constraint.
-  * Current behaviour: Two persons are identified as having the same identity if their names are the same.
-  * Enhanced behaviour: Two persons are identified as having the same identity if they have the same identification number. This can be achieved by having an `identity` field in the `Person` class. This means that two people of exactly the same name may be identified as two different persons.
-  * Justification: This allows user to add two persons of exactly the same name to the contact list. As of the current implementation, it does not allow the user to add a second person of the same name to the contact list.
-
-* More relaxed name constraint.
```

Comment:
I feel this should be kept because there are real-life examples of names that require this, e.g. "s/o". This could be seen as a functionality bug.

Reply:
Users can get around it by replacing "/" with whitespace. For comma, they can simply ignore it, or reorder the words to give a name without comma. I feel that this is a small change relative to other more important changes.

Reply:
Okay, ig if confronted, we can say that the name kept in NetworkBook does not need to match the legal name. KIV to improve explanation in UG.